tests/src/main/java/org/hibernate/beanvalidation/tck/tests/constraints/constraintdefinition/ConstraintDefinitionsTest.java

```diff
@@ -75,6 +77,36 @@ public void testConstraintWithCustomAttributes() {
 	}
 
 	@Test
+	@SpecAssertions({
+			@SpecAssertion(section = "3.2", id = "b")
+	})
+	public void testRepeatableConstraint() {
+
+		Validator validator = TestUtil.getValidatorUnderTest();
+		Set<ConstraintDescriptor<?>> descriptors = validator.getConstraintsForClass( Movie.class )
+				.getConstraintsForProperty( "title" )
+				.getConstraintDescriptors();
+
+		assertEquals( descriptors.size(), 2, "There should be two constraints on the title property." );
+		for ( ConstraintDescriptor<?> descriptor : descriptors ) {
+			assertEquals(
+					descriptor.getAnnotation().annotationType().getName(),
+					Size.class.getName(),
+					"Wrong annotation type."
+			);
+		}
+
+		Set<ConstraintViolation<Movie>> constraintViolations = validator.validate( new Movie( "A" ) );
+		assertCorrectNumberOfViolations( constraintViolations, 1 );
+
+		constraintViolations = validator.validate( new Movie( "Title" ) );
```

Comment:
Add another one showing that the second constraint is applied, too (i.e. add a title that is too short)?